desmume/src/frontend/modules/osd/agg/aggdraw.cpp

```diff
@@ -118,11 +118,7 @@ static void Agg_init_fonts()
 
 AggDraw_Desmume aggDraw;
 
-#if defined(WIN32) || defined(HOST_LINUX)
 T_AGG_RGBA agg_targetScreen(0, GPU_FRAMEBUFFER_NATIVE_WIDTH, GPU_FRAMEBUFFER_NATIVE_HEIGHT*2, 1024);
-#else
```

Comment:
shouldn't this now read `256*aggDraw.screenBytesPerPixel` instead of 1024 ?

oh, actually if bpp == 2, it should return T_AGG_RGB555 instead of T_AGG_RGBA

Reply:
You're right. The problem is it's not a function, it's global object. Made a cheap workaround. Ideally this object should be allocated dynamically, but it can severely break other frontends.